modules/ROOT/pages/customize-your-canvas.adoc

```diff
@@ -13,34 +13,27 @@ The *View* card and the canvas are updated to display the current selection.
 
 Visualizer editors can save defined views so that any Visualizer user can access them.
 When you close Anypoint Visualizer and reload the canvas later, the views you last accessed are displayed.
-If you are using a Mule runtime patch that was released after December 12, 2019, you can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers) to customize Layers, Tags, and Display Names in the Visualizer UI or by using a runtime property.
-
 
+You can customize layers, tags, and display names in the Visualizer UI or by setting a runtime property. If you are using a Mule runtime patch that was released after December 12, 2019, you can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers).
```

Comment:
... in the Architecture Visualization UI

Reply:
```suggestion
You can customize layers, tags, and display names in the Architecture Visualization UI or by setting a runtime property. If you are using a Mule runtime patch that was released after December 12, 2019, you can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers).
```